TidepoolOnboarding/Views/Accessory/Callout.swift

```diff
@@ -40,27 +42,12 @@ struct Callout<Content: View>: View {
         .overlay(RoundedRectangle(cornerRadius: 10, style: .continuous).stroke(Color(.systemFill), lineWidth: 1))
         .clipShape(RoundedRectangle(cornerRadius: 10, style: .continuous))
     }
-
-    private var warning: some View {
```

Comment:
Was this extra code?

Reply:
Yes, this private `var` was never used, so I removed it.  I'm _guessing_ it was replaced by `WarningIcon` when it was created.

Reply:
Cool, I must have missed that when I refactored `WarningIcon`.